apps/projects/templates/a4_candy_projects/includes/project_list_tile.html

```diff
@@ -1,11 +1,14 @@
 {% load i18n project_tags a4_candy_project_tags contrib_tags thumbnail rules static %}
 
 <li class="tile tile--{{ orientation }}">
-
+    <a class="tile__link" aria-labelledby="tile-title-{{ object.pk }}" href="{{ url }}">
+        <h3 class="visually-hidden" id="tile-title-{{ object.pk }}">
+            {{ object.name }}
+        </h3>
+    </a>
```

Comment:
@khamui that took me too long to spot :p the link was just inside the if for tile image, so tiles with no image didn't have it